server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/EmailParseMethod.scala

```diff
@@ -103,7 +103,7 @@ class EmailParseMethod @Inject()(val blobResolvers: BlobResolvers,
   private def parse(request: EmailParseRequest, blobId: BlobId, blobContent: => InputStream, size: Size): SMono[EmailParseResults] = {
     val result: SMono[EmailParseView] = for {
       mime4JMessage <- SMono.fromTry(Email.parseStreamAsMime4JMessage(blobContent))
-      bodyStructure <- SMono.fromTry(EmailBodyPart.of(request.bodyProperties, zoneIdProvider.get(), blobId, mime4JMessage))
+      bodyStructure <- SMono.fromTry(EmailBodyPart.ofMessage(request.bodyProperties, zoneIdProvider.get(), blobId, mime4JMessage, size))
```

Comment:
<3 cool catch thanks Rene!

Reply:
Well the Email/parse test were failing without this :)